glowing-waffles/photometry.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import print_function, division, absolute_import
+import numpy as np
+from astropy import units as u
+from astropy.stats import mad_std
+from astropy.io import fits
+from astropy.table import Table, join
+from photutils import aperture_photometry, CircularAperture, CircularAnnulus
+from photutils.background import Background
+
+
```

Comment:
can you add this line near the top here? It will make sure that if someone does `from photometry import *` the only name that will be imported is `photutils_stellar_photometry`. Not that any users should use that form of import, but it is useful in `__init__.py` for setting up the namespace of the package.